recovery/translation_tr.ts

```diff
@@ -154,42 +154,42 @@ UYARI: Bu işlem SD kartınızdaki tüm verileri silecektir</translation>
     <message>
         <location filename="initdrivethread.cpp" line="342"/>
         <source>Saving boot files to memory</source>
-        <translation type="unfinished"></translation>
+        <translation>Önyükleme dosyaları belleğe kaydediliyor</translation>
     </message>
     <message>
         <location filename="initdrivethread.cpp" line="345"/>
         <source>Error saving boot files to memory. SD card may be damaged.</source>
-        <translation type="unfinished"></translation>
+        <translation>Önyükleme dosyalarını belleğe kaydetmede hata yaşandı. SD kart zarar görmüş olabilir.<</translation>
```

Comment:
@Zaryob Somehow there was a duplicate less-than symbol (`<`) near the end of this line, which was causing problems.
I've fixed this in https://github.com/raspberrypi/noobs/commit/2f6a07e92b81e99f39cd3e274f2af663abca32aa